js/ngDialog.js

```diff
@@ -5,16 +5,16 @@
  */
 
 (function (root, factory) {
-    if (typeof module !== 'undefined' && module.exports) {
-        // CommonJS
-        module.exports = factory(require('angular'));
-    } else if (typeof define === 'function' && define.amd) {
-        // AMD
-        define(['angular'], factory);
-    } else {
-        // Global Variables
-        factory(root.angular);
-    }
+	if (typeof module !== 'undefined' && module.exports) {
+		// CommonJS
+		module.exports = factory(require('angular'));
+	} else if (typeof define === 'function' && define.amd) {
+		// AMD
+		define(['angular'], factory);
+	} else {
+		// Global Variables
+		factory(root.angular);
+	}
```

Comment:
@lelandcope that is bad, indentation shouldn't be changed - see [CONTRIBUTING](https://github.com/likeastore/ngDialog/blob/master/CONTRIBUTING.md).